secretflow/device/device/teeu.py

```diff
@@ -141,6 +143,181 @@ def run(self, func: Callable, *args, **kwargs) -> TEEUData:
         return func(*args_, **kwargs_)
 
 
+def _actor_wrapper(name, num_returns):
+    def wrapper(self, *args, **kwargs):
+        # device object type check and unwrap
+        _num_returns = kwargs.pop('_num_returns', num_returns)
+        value_flat, value_tree = jax.tree_util.tree_flatten((args, kwargs))
+        for i, value in enumerate(value_flat):
+            if isinstance(value, DeviceObject):
+                assert (
+                    value.device == self.device
+                ), f'unexpected device object {value.device} self {self.device}'
+                value_flat[i] = value.data
+        args, kwargs = jax.tree_util.tree_unflatten(value_tree, value_flat)
+
+        logging.debug(
+            (
+                f'Run method {name} of actor {self.actor_class}, num_returns='
+                f'{_num_returns}, args len: {len(args)}, kwargs len: {len(kwargs)}.'
+            )
+        )
+        handle = getattr(self.data, name)
+        res = handle.options(num_returns=_num_returns).remote(*args, **kwargs)
+        if _num_returns == 1:
+            return TEEUObject(self.device, res)
+        else:
+            return [TEEUObject(self.device, x) for x in res]
+
+    return wrapper
+
+
+def _cls_wrapper(cls):
+    def ray_get_wrapper(method):
+        def wrapper(self, *args, **kwargs):
+            logging.basicConfig(level=get_logging_level(), format=LOG_FORMAT)
+
+            logging.debug(
+                f'TEEU runs function: {method}, with args: {args}, kwargs: {kwargs}'
```

Comment:
It's not a good idea to print the specific value of parameters (it may cause log explosion), you can print the parameter length just same as other places.